style.css

```diff
@@ -478,6 +467,18 @@ nav {
   width: 100%;
 }
 
+.footer {
+  background-image: url(./image/contact\ form\ background\ shapes.svg);
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #6070ff;
+  background-repeat: no-repeat;
+  width: 100%;
+  padding: 80px 0 0 0;
+}
```

Comment:
- [ ] Kindly fix the borders of your contact form to be similar to the Figma template 👇🏽 :

-Figma template

![image](https://user-images.githubusercontent.com/58642949/186796117-ea079a91-5186-4cc4-ac00-e51015a678ca.png)

- Your design

![image](https://user-images.githubusercontent.com/58642949/186796148-fed9cac8-e0d6-4f25-af14-b760c199bd5e.png)